python_modules/dagster/dagster/core/asset_defs/decorators.py

```diff
@@ -68,6 +69,9 @@ def asset(
             If no entry is provided for a particular asset dependency, the partition mapping defaults
             to the default partition mapping for the partitions definition, which is typically maps
             partition keys to the same partition keys in upstream assets.
+        op_version (Optional[str]): (Experimental) The version of the asset's op. Op versions can
+            be used to determine whether the asset can be memoized.
+
```

Comment:
Is it correct to say that the "version can be used to determine whether the asset can be memoized"?

I'm still not that familiar with the memoization system, but I thought if it has _any_ version it _can_ be memoized-- so instead wouldn't it be better to say something like "The op version is one component of an asset's memoization key."?